UI/Pages/Index.cshtml

```diff
@@ -34,6 +34,25 @@
 				<govuk-button type="submit" data-testid="call-to-action">Continue</govuk-button>
 			</div>
 
+			@if (Model.Data.KeyStages != null)
+			{
+			    foreach (var KeyStage in Model.Data.KeyStages)
+			    {
+					<input type="hidden" id="@KeyStage" name="Data.KeyStages" value="@KeyStage" />
+				}
+			}
+			@if (Model.Data.Subjects != null)
+			{
+			    foreach (var subject in Model.Data.Subjects)
+			    {
+					<input type="hidden" id="@subject" name="Data.Subjects" value="@subject" />
+				}
+			}
+            @if (Model.Data.TuitionType.HasValue)
```

Comment:
I'm not sure that this check is necessary, The hidden fields will simply have no value if the property is null.